examples/llava/llava.cpp

```diff
@@ -2,163 +2,166 @@
 #include "llava-utils.h"
```

Comment:
The way it is now, `llava-utils.h` is both a private and public header.

Is `llava-utils.h` used in `llava.cpp`? If not, remove the include here.
If it is used, it's better to move whatever functionality is needed from `llava-utils.h` into `llava.cpp`. This way, `llava-utils.h` can remain as something similar to what `common` is to `llama.cpp` and be used only by the examples (i.e. `llava-cli` in this case).